src/System.Security.Cryptography.X509Certificates/src/Resources/Strings.resx

```diff
@@ -204,4 +204,7 @@
   <data name="WorkInProgress" xml:space="preserve">
     <value>WorkInProgress.</value>
   </data>
-</root>
+  <data name="Chain_NoPolicyMatch" xml:space="preserve">
+    <value>The certificate has invalid policy.</value>
+  </data>
+</root>
```

Comment:
\*sigh\*. Fixed. (Is there a way to make VS always write one? Opening a file in vim *just* to hit save so it fixes the last line feels like a sad use of vim.)